acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -52,9 +49,87 @@ def kill_pcp_broker(host)
   end
 end
 
+# Query pcp-broker's inventory of associated clients
+# Reference: https://github.com/puppetlabs/pcp-specifications/blob/master/pcp/inventory.md
+# @param broker hostname or beaker host object of the pcp-broker host
+# @param query pattern of client identities that you want to check e.g. pcp://*/agent for all agents, or pcp://*/* for everything
+# @return Ruby array of strings that are the client identities that match the query pattern
+# @raise Exception if pcp-client cannot connect to make the inventory request, does not receive a response, or the response is missing
+#        the required [:data]["uris"] property
+def pcp_broker_inventory(broker, query)
+  # Event machine is required by the ruby-pcp-client gem
+  # https://github.com/puppetlabs/ruby-pcp-client
+  em = Thread.new { EM.run }
```

Comment:
Needs to pause until EM is started to get the benefit of this:

https://github.com/faye/faye-websocket-ruby/blob/master/lib/faye/websocket.rb#L37

Will be updating the example usage in ruby-pcp-client as part of the rework I'm doing.